README.md

```diff
@@ -103,26 +103,30 @@ W razie problemów z uruchomieniem projektu utwórz [nowe zagadnienie](https://g
 
 ## Testy automatyczne
 
-Projekt wykorzystuje testy automatyczne, które zapewniają weryfikacje wszystkich wprowadzonych zmian. Wszelkie proponowane zmiany z wykorzystaniem [GitHub Actions](https://github.com/watchdogpolska/small_eod/actions).
+Projekt wykorzystuje testy automatyczne, które zapewniają weryfikacje wszystkich wprowadzonych zmian. Sa one uruchamine w środowisku [GitHub Actions](https://github.com/watchdogpolska/small_eod/actions).
 
-Wszelkie zmiany w repozytorium będą uruchamiały serie testów automatycznych, ale przed ich dodaniem masz możliwość wykonania testów lokalnie za pomocą [Makefile](https://en.wikipedia.org/wiki/Makefile).
-
-W celu wykonanie testów automatycznych formatowania należy wykonać:
+W celu wykonania testów automatycznych backendu należy wykonać:
 
 ```bash
-make lint
+make test-django-backend
 ```
 
-Większość problemów w formatowania można naprawić z wykorzystaniem automatycznego formatowania za pomocą wykonania:
+## Statycnza kontrola kodu
+
+Statycznaa kontrolu kodu w projekcie służy do sprawdzenia, czy kod spełnia określone standardy jakości. Wszystkie sprawdzenia kodu statycznego można uruchamiać za pomocą [pre-commit run](https://pre-commit.com/).
+Sa one weryfikowane w środowisku [GitHub Actions](https://github.com/watchdogpolska/small_eod/actions).
+
+W celu wykonanie kontroli statycznej kodu należy wykonać:
 
 ```bash
-make fmt
+pre-commit run --all-files
 ```
 
-W celu wykonania testów automatycznych backendu należy wykonać:
+To turn on pre-commit checks for commit operations in git, enter:
```

Comment:
Ja wiem, że miasza nam się polski z angielskim w pull requestach i issues, ale czy moglibyśmy w README trzymać się polskiego?

Reply:
plus to zdanie nie za bardzo ma sens w swoim kontekście

Reply:
Ja dokumentacje biznesową i techniczną proponuje prowadzić w języku polskim, aby uczynić ją odrobinę dostępniejszą dla biznesu, a także nowych osób.